go.mod

```
@@ -3,7 +3,7 @@ module github.com/contentsquare/chproxy
 go 1.13
```

I think you should bump the version to the supplied in docker image:
```suggestion
go 1.17
```
You'll have to run after that 
```
go mod tidy 
```.
Tested in local, works fine